Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -465,6 +465,29 @@ ShaderCode GenerateVertexShaderCode(APIType api_type, const vertex_shader_uid_da
   // get rasterized correctly.
   out.Write("o.pos.xy = o.pos.xy - o.pos.w * " I_PIXELCENTERCORRECTION ".xy;\n");
 
+  if (g_ActiveConfig.bVertexRounding)
```

Comment:
you also should be checking for 1x IR here, because ini files bypass the check in the configuration screen